### DISCUSSION TITLE
Extracting pdf data to an excel

### DISCUSSION THREAD

**Discussion by Nicodiazsanf** [jsvine/pdfplumber#644] [Ask for help with specific PDFs]
Dear community, 

How can I export the information of the annexed invoice  to an excel? The information that I really need is everything on the "DETALLE" column, and the amount of "MONTO NETO and "MONTO TOTAL"

I've tried with this code, but I have an error with the extract_text().... ValueError: not enough values to unpack (expected 2, got 1)
Maybe the reason is the image at the begging of the invoice... 

_import pdfplumber
with pdfplumber.open('/Users/nicolasdiaz/Desktop/MARFIL/Invoices/77116757-T33-F1849.pdf') as pdf:
    page = pdf.pages[0]
    text = page.extract_text()
print(text)_

[77116757-T33-F1849.pdf](https://github.com/jsvine/pdfplumber/files/8526647/77116757-T33-F1849.pdf)

**Comment by jsvine** [comment]
Hi @Nicodiazsanf, and thanks for your interest in this library. The issue you've encountered *as well as the solution* appear to be the same as in Discussion https://github.com/jsvine/pdfplumber/issues/636#issuecomment-1095042549 

On my computer, making that fix works for me. If it doesn't work for you, let me know.
